include/llbuild/Basic/Tracing.h

```diff
@@ -47,17 +47,17 @@ static os_signpost_id_t signpost_id() {
 
 /// Begin an interval if tracing is enabled.
 #define LLBUILD_TRACE_INTERVAL_BEGIN(name, ...) { \
-if (__builtin_available(macOS 10.14, *)) os_signpost_interval_begin(getLog(), signpost_id(), name, ##__VA_ARGS__); \
+if (__builtin_available(macOS 10.14, iOS 12.0, *)) os_signpost_interval_begin(getLog(), signpost_id(), name, ##__VA_ARGS__); \
```

Comment:
Can you add tvOS/watchOS to these three as well? There's also a __builtin_available in adjust-times.cpp that could use adjustment.

Reply:
OK, can do -- I'll also set TVOS_DEPLOYMENT_TARGET to 9.0 and WATCHOS_DEPLOYMENT_TARGET to 2.0 to match the iOS deployment target when I fix this.